edx_event_bus_kafka/internal/tests/test_consumer.py

```diff
@@ -239,11 +238,13 @@ def fake_emit(*args, **kwargs):
         ],
         key=b'\x00\x00\x00\x00\x01\x0cfoobob',  # Avro, as observed in manual test
         value=b'XXX',
-        error=TEST_KAFKA_ERROR,  # Add the error, in case we use this at some point
```

Comment:
**Note to reviewer**: We can make the test data and tests more complicated if and when we ever rely on this.

Please mark as resolved when done reading, if you have no response. Thank you.